src/containers/StructurePage/folderComponents/menuOptions/AddExistingSubjectTopic.jsx

```diff
@@ -37,23 +41,45 @@ class AddExistingSubjectTopic extends React.PureComponent {
     const subjectTopics = await fetchSubjectTopics(subjectId);
 
     this.setState({
-      topics: topics.filter(
-        topic => !subjectTopics.some(t => t.id === topic.id),
-      ),
+      topics: topics
+        .filter(topic => !subjectTopics.some(t => t.id === topic.id))
```

Comment:
Her fjerner du alle emner som finnes i faget du står i. Det burde være mulig å flytte emner innad i fag også.

Reply:
Når eg tenker meg om så sikrer dette at vi ikkje får sirkulære avhengigheter. Dersom vi tillater flytting innad i fag kan vi risikere at et emne legges som underemne til seg sjølv. Vi lar det være slik. Då må heller flytting innad i fag være en tostegsflytting.

Reply:
Ja det er sant. Jeg kan fjerne det for SubjectTopic, for det vil vel alltid være greit å legge til et underemne til toppnivået på faget? Endrer og navn til AddExsistingToSubjectTopic, for å gjøre det litt mindre forvirrende

Reply:
👍